libafl/src/monitors/tui/ui.rs

```diff
@@ -63,15 +63,19 @@ impl TuiUI {
 
     pub fn on_right(&mut self) {
         // never 0
-        self.clients_idx = 1 + self.clients_idx % (self.clients - 1);
+        if self.clients != 0 {
```

Comment:
The `never 0` comment above is confusing

Reply:
Yes, I was confused about that comment too. I am not sure what it is supposed to mean, but I think `self.clients_idx` is never 0.

Reply:
Maybe move the comment into the if, then at least it's commenting the same line?